tooling.json

```diff
@@ -104,19 +104,23 @@
             }
         },
         "fair": {
-            "fair-evaluator": {
-                "docs": "https://github.com/EOSC-synergy/FAIR_eva",
+            "fair-eva.py": {
```

Comment:
just a comment (it is ok like that): if you want to preserve `fair-evaluator` key for this entry, you can do so and then use `"executable": "fair-eva.py"` to point to the script.